app/javascript/shared/components/__tests__/tags.test.jsx

```diff
@@ -20,7 +20,7 @@ describe('<Tags />', () => {
   });
 
   describe('handleKeyDown', () => {
-    it('does not call preventDefault on used keyCode', () => {
+    it.skip('does not call preventDefault on used keyCode', () => {
```

Comment:
I don't understand _why_ this is failing, but the reported error is

```
 RUNS  app/javascript/shared/components/__tests__/tags.test.jsx
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "FetchError: invalid json response body at  reason: Unexpected end of JSON input".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

Setting `const tests = [];` also permits the test to pass (by not pressing any keys, nothing is erroring), as well as the three keys other than `','`, 'a', '1', and 'Enter' process successfully.

This suggests something might have broken in the handling of the comma character in the tag list field?

Reply:
@djuber, I figured out why it fails. In this scenario, when the comma is pressed (simulated), it calls this chunk of code in shared/components/tags.jsx

https://github.com/forem/forem/blob/2226398f73f3883fa3de8424d5ee614c9357aa2d/app/javascript/shared/components/tags.jsx#L319-L327

Then call fails hence the `UnhandledPromiseRejection `. I'm going to update the test to fix this.

Reply:
Actually, I’m going to put up a PR for this test separately. It was validly failing before but silently. I believe the v16 upgrade of Node.js just has it popping as an `UnhandledPromiseRejection`